generators/hls-example/run_hls.tcl

```diff
@@ -0,0 +1,8 @@
+open_project -reset proj_gcd_example
+add_files accel/HLSAccel.cpp
+set_top HLSAccelBlackBox
+open_solution -reset "solution1"
+set_part {xcvu9p-flgb2104-2-i}
+create_clock -period 10
```

Comment:
I see that both the FPGA part number and the clock period should probably configurable from the makefile and piped to the Chisel.